Address data

ShahrajKarim opened the pull request:
Obtained postcode data for addresses in Leeds.
Investigated vacant properties at the LSOA level.

ShahrajKarim committed:
Address data merging using council tax data, uprn data and epc data.

ShahrajKarim committed:
Investigated vacant properties at the LSOA level and plotted a histogram for distribution. Will make an Rmd file in the future.

ShahrajKarim committed:
Rmd file

ShahrajKarim committed:
Remove vacant property R file from branch - replaced with Rmd and html

lasse-meixner reviewed (approved):
Looks good. So we can explain about 50% of the additional addresses with vacant properties, do we have any theories where the rest of the discrepancy comes from?

ShahrajKarim commented:
> Looks good. So we can explain about 50% of the additional addresses with vacant properties, do we have any theories where the rest of the discrepancy comes from?

I have a few theories, let me know :

1. The census response rate by LAD was between 89 and 99 percent. I think that some households may not have been recorded as a result and this could partially explain the difference as wel.

2. There were some business addresses that were in the council tax data e.g. Travelodge, Holiday Inn. I'm quite sure that these addresses are not in the final set of address data though, so this is likely to be a dead end.

3. It's possible for some postcodes to fall within two LAD's. Since the household level data was collected at the LSOA level, some census respondents may be incorrectly assigned to a different LAD instead of Leeds. I doubt that this is a major issue though.

lasse-meixner commented:
Yea 1 could totally explain it. So the idea is that these households that did not respond in the census may not show up in the census count at all?
Re 2 - are you trying to filter those businesses  out?

ShahrajKarim commented:
> Yea 1 could totally explain it. So the idea is that these households that did not respond in the census may not show up in the census count at all? Re 2 - are you trying to filter those businesses out?

Yep that's exactly what I'm thinking for point 1.

For point 2, I did have a look into it by searching up known hotel chains such as the ones I mentioned before, but they are no longer in the dataset. I did search up the term "hotel" as well and only 8 properties showed up. I think that since the tax dataset is specifically for residential properties the only businesses we should find are ones such as hotels.

ShahrajKarim committed:
Changed 50% to 45%

ShahrajKarim committed:
Fixed issue where properties with door_number and cleaned_addr1 "NA" were merged together incorrectly.

Also, imputed some missing postcodes using street name.

ShahrajKarim committed:
deleted vacant_property_analysis files

ShahrajKarim committed:
Fixed errors related to renaming file

ShahrajKarim committed:
Fixed an error related to addr1 of some hotels

ShahrajKarim committed:
Created Rmd file exploring the address mismatch.
These include vacant properties and also unconvential residential addresses e.g. hotels, inns and student accommodations.

ShahrajKarim committed:
Made hotel distribution easier to visualise by excluding LSOA counts equal to 0

ShahrajKarim committed:
Fixed issue with Rmd files

ShahrajKarim committed:
Explored mismatch purely from unconventional residential addersses.

ShahrajKarim committed:
tagged addresses which households do not live in e.g. vacant properties, hotels, student acommodations
